README.md

```diff
@@ -71,22 +71,22 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+this definitely seems like a model method to me. separation of concerns.
 ```
```

Comment:
This would work best as a helper method, since it determines how things are rendered in the browser (i.e., it affects the HTML). We use model methods when dealing with data coming in and out of the database.